WinApi/Gdiplus/Helpers.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.IO;
+
+namespace WinApi.Gdiplus
+{
+    public static class GdiplusHelpers
+    {
+        public static byte[] CreateBitmap(IntPtr hGlobal)
```

Comment:
since this method generates an image NOT in memory, maybe return the image itself(or filename, a handle, ect) and not bytes?